api/wfx.go

```diff
@@ -36,8 +37,10 @@ const (
 )
 
 type WfxServer struct {
-	storage persistence.Storage
-	checker health.Checker
+	storage          persistence.Storage
+	checker          health.Checker
+	ssePingInterval  time.Duration
```

Comment:
Nitpick: Shouldn't this be a `struct sse` containing the two configurations? They are just two but anyway...